src/Api/rewards.js

```diff
@@ -1,15 +1,14 @@
 import { ARBITRUM, ARBITRUM_TESTNET } from "../Helpers";
 
-const BASE_TRACER_URL = 'https://dev.api.tracer.finance'
+const BASE_TRACER_URL = "https://dev.api.tracer.finance";
 
-
-export function getTracerServerUrl(chainId, path) {
+export function getTracerServerUrl(chainId, path, params) {
   if (!chainId) {
     throw new Error("chainId is not provided");
   }
   if (chainId === ARBITRUM_TESTNET) {
-    chainId = ARBITRUM; 
+    chainId = ARBITRUM;
   }
 
-  return `${BASE_TRACER_URL}/trs${path}?network=${chainId}`;
+  return `${BASE_TRACER_URL}/trs/${path}?network=${chainId}${params ? `&${params}` : ""}`;
```

Comment:
the params should be seperate from here, you can pass them in with useSWR https://swr.vercel.app/docs/arguments think there is an example somewhere

Reply:
Also changing this to include the 'trs/' will break use else where. Would be good to change back to 
```suggestion
  return `${BASE_TRACER_URL}/trs${path}?network=${chainId}`;
```